src/extension/inlineEdits/vscode-node/parts/vscodeWorkspace.ts

```diff
@@ -381,14 +381,45 @@ export class VSCodeWorkspace extends ObservableWorkspace implements IDisposable
 	}
 }
 
-export interface IVSCodeObservableTextDocument extends IObservableDocument {
-	kind: 'textDocument';
-	readonly textDocument: TextDocument;
+
+export interface IBaseVSCodeObservableDocument extends IObservableDocument {
```

Comment:
@hediet This is still part of a few other PRs.
Eventually I will rename `IBaseVSCodeObservableDocument` into `IVSCodeObservableDocument`